src/numberDisplayFormat.ts

```diff
@@ -204,7 +204,10 @@ function getCommifiedFormat(
   }
   if (actualValue === '') return '0'
   const actualValueWAD = getWad(actualValue)
-  if (actualValueWAD.isZero()) return getDpFormat('0.0', decimalPlaceInput)
+  if (actualValueWAD.isZero())
+    return decimalPlaceInput === 0
+      ? '0.00'
```

Comment:
Case 1: if `actualValue` is `0.000` and `decimalPlace` is `0`, the result will be `0`.
Case 2: if `actualValue` is `0.000` and `decimalPlace` is `exact`, the result will be `0.000`.
Case 3: if `actualValue` is `0.0000` and `decimalPlace` is `exact`, the result will be `0.0000`.

In your logic, it will always be `0.00`.